lib/matplotlib/backends/backend_ps.py

```diff
@@ -1239,6 +1250,9 @@ def xpdf_distill(tmpfile, eps=False, ptype='letter', bbox=None, rotated=False):
     operators. This distiller is preferred, generating high-level postscript
     output that treats text as text.
     """
+    mpl._get_executable_info("gs")
```

Comment:
Should this check be up around line 1218?

Reply:
No, this is for ps2pdf (which is actually just a thin wrapper script around gs).  Added a comment clarifying.

Reply:
Why doesn't `gs_distill` need an equivalent check?

Reply:
It already calls `mpl._get_executable_info("gs")` basically at the beginning.

Reply:
Sorry, I was looking for it on it's own line an not parsing that it was called in-line to another function call :sheep: